service/lib/agama/dbus/manager.rb

```diff
@@ -56,13 +56,18 @@ def initialize(backend, logger)
       CONFIG_PHASE = 1
       INSTALL_PHASE = 2
 
+      IGUANA_NOT_USED = false
```

Comment:
np: Perhaps this is too much... what about using true/false?

Reply:
removed as underlying ```backend.iguana?``` returns true/false anyway ... this was just leftover of good old mocking times ;-)